data/json/items/comestibles.json

```diff
@@ -4324,44 +4324,335 @@
     "charges": 2,
     "fun": 2
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "almonds",
+    "name": "almonds",
+    "name_plural": "almonds",
+    "weight": 32,
+    "color": "brown",
+    "spoils_in": 8640,
+    "container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -3,
+    "healthy": 2,
+    "calories": 80,
+    "description": "An imported bag of almonds.",
```

Comment:
You don't have to say they are imported. America is the number one producer of almonds in the world.

Reply:
Adjustment made, as well as added tree for almonds for use in orchards.